packages/db/src/translate/reconciliation_reports.ts

```diff
@@ -0,0 +1,23 @@
+import { PostgresReconciliationReportRecord } from '../types/reconciliation_report';
+
+/**
+ * Generate an API Reconciliation Report record from a PostgreSQL record.
+ *
+ * @param {Object} pgReconciliationReport - a PostgreSQL reconciliation report record
+ * @returns {Object} an API reconciliation report record
+ */
+export const translatePostgresReconciliationReportToApiReconciliationReport = (
+  pgReconciliationReport: PostgresReconciliationReportRecord
+) => {
+  const apiReconciliationReport = {
+    // id or cumulus_id?
```

Comment:
?

Reply:
I'm not as familiar with the translate functions, and what the api type records require. I saw an id in the async_operations and provider translate files, but not the others. Everything works here so far without an id attribute, but I wasn't sure if it might be needed later in some way and thought others might know better.

Reply:
Yeah, this is where I wish we had a more rigid schema for the API returns that we could compare against.

I very strongly suspect it's not cumulus_id and that's an internal field at this point not returned generally by the API (or maybe ever). The table doesn't have another ID similar to granuleId or collectionId. Just `name` as the unique identifier. If that matches what dynamo was doing, that's probably sufficient.